.github/workflows/auto_update_doc.yml

```diff
@@ -33,6 +37,14 @@ jobs:
       run: |
         python -m pip install -q --upgrade pip
         python -m pip install -r requirements-release.txt
+        #apt-get install libprotobuf-dev protobuf-compiler
+
+        wget https://github.com/protocolbuffers/protobuf/releases/download/v25.1/protoc-25.1-linux-x86_64.zip 
+        unzip protoc-4.25.1-linux-x86_64.zip -d protoc
```

Comment:
Why is this needed?

Reply:
The idea was to use a precompiled protobuf 4 compiler. This is not available via apt-get in ubuntu-24.04. But it's not currently running.
@cyyever What doesn't fit yet?

Reply:
Which job?

Reply:
Thanks for the hint: I included the build script. Should work now.

Reply:
Protobuf is installed in non-standard location

Reply:
I wonder why the pipeline works in the separate new job sdist_test.yml, but not in the auto_update_doc.yml ?

Reply:
I think we should just use the default one downloaded by cmake. There shouldn’t be any extra step to set up protobuf right?

The idea is that we will just do pip install . and nothing else to get the package ready.

Reply:
If we don't use ``pip install -r requirements-release.txt`` in this CI job and instead we use ``pip install -r requirements.txt``, it should work.

Reply:
I think we should start adding the comments at the point to which they refer, otherwise the assignment to the lines is not always clear to me.

I would close "this" conversation now.